src/DotLiquid/Context.cs

```diff
@@ -389,6 +389,18 @@ private object Resolve(string key, bool notifyNotFound = true)
                 case "false":
                     return false;
                 case "blank":
+                    return new Symbol(o => {

```

Comment:
Try to use full English words when possible for variable names instead of single letter o, b, s, etc.